File: Difficulty: Basic/Remove Spaces/remove-spaces.cpp
class Solution {
  public:
    string removeSpaces(string& s) {
        // code here
        int n=s.size();
        string result= "";
        for(int i=0;i<n;i++){
            if(s[i]!=' '){
                result+= s[i]; 
            }
        }
        return result;
    }
};

File: Difficulty: Basic/Remove Spaces/remove-spaces.java
class Solution {
    public String removeSpaces(String s) {
        // code here
        int n=s.length();
        String str="";
        for(int i=0;i<n;i++){
            if(s.charAt(i)!=' '){
                str+=s.charAt(i);
                
            }
        }
        return str;
    }
}